components/popover_list_members/popover_list_members.jsx

```diff
@@ -50,12 +52,15 @@ export default class PopoverListMembers extends React.Component {
         $('.member-list__popover .popover-content .more-modal__body').perfectScrollbar();
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
-        if (this.props.users !== nextProps.users || this.props.statuses !== nextProps.statuses) {
-            const sortedUsers = this.sortUsers(nextProps.users, nextProps.statuses);
-
-            this.setState({sortedUsers});
+    static getDerivedStateFromProps(nextProps, prevState) {
+        if (nextProps.users !== prevState.users || nextProps.statuses !== prevState.statuses) {
+            return {
+                users: nextProps.users,
+                statuses: nextProps.statuses,
+                sortUsers: this.constructor.sortUsers(nextProps.users, nextProps.statuses),
```

Comment:
I don't think `this` is defined here, but if it is, it feels weird to call out to that, and I imagine it might break at some point. Could you either make it into a static method like this one or just call `Utils.sortUsersByStatusAndDisplayName` directly here?